CONTRIBUTING.md

```diff
@@ -38,38 +38,39 @@ The Makefile takes care of installing the required toolchain as needed. The
 toolchain and other build related files are cached under the `.build` folder
 (ignored by git).
 
-## Development in Docker
+### Development in Docker
 
 You can either build SPIRE on your host or in a Ubuntu docker container. In
 both cases you will use the same Makefile commands.
 
 To build SPIRE within a container, first build the development image:
 
-```
-$ make dev-image
```

Comment:
The inclusion of this character historically indicates whether the user needs to be root to run the command or not. Which lint rule does it violate?

Reply:
it was affected by [MD014](https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md).
That prevents to use "$" if we does not provides an output

Reply:
I think I'd rather see that disabled. I agree with evan that `$` and `#` are important visual distinctions.

Reply:
about disable this one, do you prefer I disable it for all   files, or only this one?
we can solve this just adding an output to that example

Reply:
I'm ok if we want to add output if it makes sense, but in general, I'm don't know if I agree with this rule. I don't see the benefit.

Reply:
ok, I'll just disable this on linter